tests/functional/test_request_pages.py

```diff
@@ -104,3 +104,104 @@ def test_request_group_edit_comment(live_server, context, page, bll, settings):
 
     comments_locator = contents.locator(".comments")
     expect(comments_locator).to_contain_text("test comment")
+
+
+def test_request_return(live_server, context, page, bll):
+    author = login_as_user(
+        live_server,
+        context,
+        user_dict={
+            "username": "author",
+            "workspaces": ["workspace"],
+            "output_checker": False,
+        },
+    )
+
+    release_request = factories.create_release_request(
+        "workspace",
+        user=author,
+    )
+    factories.write_request_file(
+        release_request,
+        "group",
+        "file1.txt",
+        "file 1 content",
+    )
+    factories.write_request_file(
+        release_request,
+        "group",
+        "file2.txt",
+        "file 2 content",
+    )
+    factories.bll.set_status(release_request, RequestStatus.SUBMITTED, author)
+
+    return_request_button = page.locator("#return-request-button")
+
+    def _review_files(username):
```

Comment:
this is nice pattern, we should maybe think about a formal helper in future, as I think it will be useful in a bunch of functional tests.